### DISCUSSION TITLE
Make /tps redirect to /spark tps

### DISCUSSION THREAD

**Discussion by rwnd-bradley** [PaperMC/Paper#11125] [Ideas]
### Is your feature request related to a problem?

With the recent bundling of spark, I think it would be beneficial for the /tps command to redirect to /spark tps for a more informative output. This was mentioned in the closed discussion #11123, unsure on whether it was rejected. 

### Describe the solution you'd like.

/tps to output /spark tps

### Describe alternatives you've considered.

N/A

**Comment by powercasgamer** [comment]
This is mentioned in https://github.com/PaperMC/Paper/discussions/10565#discussioncomment-10103482

**Reply by rwnd-bradley** [reply]
My bad
